doc/example/amici.ipynb

```diff
@@ -907,13 +1018,16 @@
    "execution_count": null,
    "metadata": {
     "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
     "pycharm": {
      "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
-    "n_starts = 20  # usually a value >= 100 should be used\n",
+    "n_starts = 10  # usually a value >= 100 should be used\n",
```

Comment:
I would just check that these 10 starts with the set random seed give a good model fit. It's kinda important to have fast and good sampling and profiling.

Also, is 10 enough to have a parameter histogram and scatter plot that looks like something nice?

Reply:
think problem would here be more that we do find optima veeery well and hence the correlation plot looks hideous. You think we should switch back to boehm? Histogram looks soso

Reply:
Yeah, if it makes the notebook, the figures and conclusions taken from them make more sense, I would go for that rather than it being a bit faster